src/main/java/org/mifos/connector/gsmastub/api/AccountsApiController.java

```diff
@@ -22,6 +23,7 @@
 
 @javax.annotation.Generated(value = "io.swagger.codegen.v3.generators.java.SpringCodegen", date = "2022-09-27T11:46:46.417Z[GMT]")
 @RestController
+@Hidden
```

Comment:
@fynmanoj @avikganguly01  should we add this type of annotation is interface or its implementation?